app/views/notification_mailer/info_request_batches/messages/_overdue.text.erb

```diff
@@ -0,0 +1,21 @@
+<%= n_('{{count}} request has not had a prompt response:',
+       '{{count}} requests have not had prompt responses:',
+       notifications.size,
+       count: notifications.size) %>
+
+<%= n_('You can see the request and remind the body to respond with the ' \
+       'following link:',
+       'You can see the requests and remind the bodies to respond with the ' \
+       'following links:',
+       notifications.size) %>
+
+<% notifications.each do |notification| %>
+  <% info_request = notification.info_request_event.info_request %>
+  <% post_redirect = PostRedirect.create!(
+       :uri => respond_to_last_url(info_request, :anchor => "followup"),
+       :user_id => info_request.user.id) %>
+  <% confirm_url = confirm_url(:email_token => post_redirect.email_token) %>
```

Comment:
I copied what the existing emails had here so as not to change too much in one PR. The alternative that we tried last time also has some issues: https://github.com/mysociety/alaveteli-professional/issues/288 and the best solution I can think of there will require deeper changes to the sign in process, so I think this is better solved separately?